config/csm/rest_response_data.yaml

```diff
@@ -43,17 +44,27 @@
     - "Max_buckets: Not a valid integer."
     - "Suspended: Not a valid boolean."
     - "Op_mask: Invalid value."
+  MalformedRequest:
+    - "{}: Must be one of: {}"
```

Comment:
Start using string template for externalized strings or messages . The references of string format is insecure and comes in critical category.

Reply:
updated